test/usub-sat.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-unknown"
+
+; Function Attrs: nounwind readnone
+define spir_func i32 @Test(i32 %x) local_unnamed_addr #0 {
+entry:
+  %0 = call i32 @llvm.usub.sat.i32(i32 3, i32 %x)
+  ret i32 %0
+}
+
+; CHECK: Constant {{[0-9]+}} [[three:[0-9]+]] 3
+; CHECK: Constant {{[0-9]+}} [[zero:[0-9]+]] 0
+
+; CHECK: Function
+; CHECK: FunctionParameter {{[0-9]+}} [[x:[0-9]+]]
+; CHECK: ISub {{[0-9]+}} [[sub_res:[0-9]+]] [[three]] [[x]]
+; CHECK: UGreaterThan {{[0-9]+}} [[cmp_res:[0-9]+]] [[three]] [[x]]
+; CHECK: Select {{[0-9]+}} [[sel_res:[0-9]+]] [[cmp_res]] [[sub_res]] [[zero]]
+
```

Comment:
Could you please also check that `[[sel_res]]` is used as a return value?

Reply:
Ok, done.